Whisper.net/LibraryLoader/NativeLibraryLoader.cs

```diff
@@ -40,14 +56,17 @@ internal static LoadResult LoadNativeLibrary()
             _ => throw new PlatformNotSupportedException($"Unsupported OS platform, architecture: {RuntimeInformation.OSArchitecture}")
         };
 
-        var assemblySearchPath = new[]
+        if (string.IsNullOrEmpty(path))
         {
-            AppDomain.CurrentDomain.RelativeSearchPath,
-            Path.GetDirectoryName(typeof(NativeMethods).Assembly.Location),
-            Path.GetDirectoryName(Environment.GetCommandLineArgs()[0])
-        }.Where(it => !string.IsNullOrEmpty(it)).FirstOrDefault();
+            var assemblySearchPath = new[]
+            {
+                AppDomain.CurrentDomain.RelativeSearchPath,
+                Path.GetDirectoryName(typeof(NativeMethods).Assembly.Location),
+                Path.GetDirectoryName(Environment.GetCommandLineArgs()[0])
+            }.Where(it => !string.IsNullOrEmpty(it)).FirstOrDefault();
 
-        var path = Path.Combine(assemblySearchPath, "runtimes", $"{platform}-{architecture}", $"whisper.{extension}");
+            path = Path.Combine(assemblySearchPath, "runtimes", $"{platform}-{architecture}", $"whisper.{extension}");
+        }
 
         if (defaultLibraryLoader != null)
         {
```

Comment:
Currently, if someone will just update the library to the latest version, they will get FilenotFoundException:
`Native Library not found in path {somePath}\runtimes\win-x64\whisper.dll. Probably it is not supported yet`

Let's add some info hint that they might need to install Whisper.net.Runtime. What do you think?

Ofc, we'll also write in the release notes that this will be a breaking change. but sometimes people are not reading those.

Reply:
https://github.com/sandrohanea/whisper.net/pull/30/commits/f66101e298d3d710c13cd8ec860198b50a8dcecd

I'm unsure of the best way to phrase it; if you can think of something better, I'm open to it.

Reply:
I think it's pretty clear how you wrote it. Thank you!